content/compose/file-watch.md

```diff
@@ -59,7 +59,7 @@ by the configured user, use the `COPY --chown` flag:
 
 ```dockerfile
 # Run as a non-privileged user
-RUN adduser -ms /bin/bash -u 1001 app
+RUN useradd -ms /bin/bash -u 1001 app
```

Comment:
As this is a generic example, perhaps we should look which one is more portable. For example, `adduser` is available in minimal images, such as `busybox` (and `alpine`, which ships with a busybox userland).

If we want it to be more portable, we should also avoid using `/bin/bash` for the example (which is not available in many images), and `/bin/sh` instead.

(Perhaps change the example to be a complete dockerfile, including a `FROM` lins so that it can actually be run).


@dvdksn @glours wdyt?

Reply:
isn't `useradd` the "canonical" command?

Reply:
Agree that it would be nice to have full Dockerfile. `/bin/sh` is fine for me. Arguably `/usr/bin/env sh` is technically more correct :p